src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -22,6 +22,10 @@
 
 class CountryType extends AbstractType
 {
+    public const FORMAT_WITH_NAME = 'format_with_name';
+    public const FORMAT_WITH_FLAG = 'format_with_flag';
+    public const FORMAT_WITH_FLAG_AND_NAME = 'format_with_flag_and_name';
```

Comment:
I think the idea for these constants was to contain the format with placeholders directly:

```suggestion
    public const FORMAT_WITH_NAME = '{name}';
    public const FORMAT_WITH_FLAG = '{flag}';
    public const FORMAT_WITH_FLAG_AND_NAME = '{flag} {name}';
```